ambari-server/src/main/resources/Ambari-DDL-Postgres-CREATE.sql

```diff
@@ -125,8 +125,10 @@ CREATE TABLE servicegroups (
   id BIGINT NOT NULL,
   service_group_name VARCHAR(255) NOT NULL,
   cluster_id BIGINT NOT NULL,
+  stack_id VARCHAR(255) NOT NULL,
   CONSTRAINT PK_servicegroups PRIMARY KEY (id, cluster_id),
-  CONSTRAINT FK_servicegroups_cluster_id FOREIGN KEY (cluster_id) REFERENCES clusters (cluster_id));
+  CONSTRAINT FK_servicegroups_cluster_id FOREIGN KEY (cluster_id) REFERENCES clusters (cluster_id),
+  CONSTRAINT UQ_servicegroups_stack_id UNIQUE (stack_id));
```

Comment:
Should be FK'ing to the stack table for stack_id, not using a unique string.

Reply:
Agreed this should be a FK reference to stack_id in the stacks table.